classes/Tools.php

```diff
@@ -1859,6 +1859,9 @@ public static function toUnderscoreCase($string)
 
 	public static function getBrightness($hex)
 	{
+		if ($hex == 'transparent')
```

Comment:
Hi,

Thank you for your PR, maybe you can use "if (Tools::strtolower($hex) == 'transparent') " ?

Best regards.